### ISSUE TITLE
Performance of FlowNet2-CSS-ft-sd

### ISSUE THREAD

**Issue by yxqlwl** [lmb-freiburg/flownet2#13]
After testing FlowNet2-CSS-ft-sd on sintel training dataset, I found that:

1. On sintel clean, FlowNet2-CSS-ft-sd can produce exact the same AEE (2.08) as reported in <FlowNet 2.0: Evolution of Optical Flow Estimation with Deep Networks>;

2. However, AEE on sintel final is 3.57, which is higher than the result reported in the paper.

So could you please check the performance of FlowNet2-CSS-ft-sd on sintel final? Thanks~

**Comment by brjeon** [comment]
Hi yxqlwl,
Have you performed training also with the provided prototxt?

**Comment by yxqlwl** [comment]
Hi,
    No I didn't. I only used the provided deploy.prototxt and caffemodel.

**Comment by yxqlwl** [comment]
Has anyone also tested performance of the models on sintel final?

**Comment by nikolausmayer** [comment]
Hi,
We will look into this.

Best,
Nikolaus

**Comment by eddy-ilg** [comment]
Thanks for the comment, will check it this week.

**Comment by eddy-ilg** [comment]
We confirm this issue. We originally made a dataset for sceneflow, where we took the sintel flow and disparity datasets together. The CLEAN images of both datasets are exactly the same, however the disparity FINAL images turn out to come without the atmospheric effects. I.e. we tested on Sintel with motion blur but missing the effects like fog.

We apologize for the inconvinience and will fix the numbers in the paper.
